packages/p2p/src/pages/orders/chat/chat-footer/chat-footer.tsx

```diff
@@ -2,19 +2,20 @@ import React from 'react';
 import classNames from 'classnames';
 import { Input, Text } from '@deriv/components';
 import { isMobile } from '@deriv/shared';
-import PropTypes from 'prop-types';
-import { observer } from 'mobx-react-lite';
+import { observer } from '@deriv/stores';
 import { localize, Localize } from 'Components/i18next';
-import ChatFooterIcon from 'Pages/orders/chat/chat-footer-icon.jsx';
 import { useStores } from 'Stores';
-import ChatMessage from 'Utils/chat-message';
-import './chat-footer.scss';
+import { handleCtrlEnterKeyPressed } from 'Utils/chat-message';
+import ChatFooterIcon from './chat-footer-icon';
+
+type TElement = HTMLInputElement | HTMLTextAreaElement;
+type TChangeEvent = React.ChangeEvent<TElement> | { target: TElement };
 
 const ChatFooter = observer(() => {
     const { order_store, sendbird_store } = useStores();
-    const file_input_ref = React.useRef(null);
-    const text_input_ref = React.useRef(null);
-    const [character_count, setCharacterCount] = React.useState(0);
+    const file_input_ref = React.useRef<TElement | null>(null);
+    const text_input_ref = React.useRef<TElement | null>(null);
+    const [character_count, setCharacterCount] = React.useState<number>(0);
```

Comment:
```suggestion
    const [character_count, setCharacterCount] = React.useState(0);
```

Don't need to define this, since the initial value of this is number so the type would be number by default!

Reply:
oh.. so its mandatory only in case of objects and array values? @niloofar-deriv

Reply:
No we don't need that, we just need to define types whenever we don't have initial values or other special situations like we are passing undefined as a initial value or ...